edx_rest_api_client/tests/mixins.py

```diff
@@ -1,7 +1,7 @@
 import responses
 
 
-class AuthenticationTestMixin(object):  # pylint: disable=useless-object-inheritance
```

Comment:
@jmbowman: I know this is a little weird because quality was still run under Python 3 (according to tox), but maybe it is due to pinning to older lint rules?